.tekton/main-pull-request.yaml

```diff
@@ -299,6 +299,22 @@ spec:
           image: registry.access.redhat.com/ubi8/ubi:latest
           script: exec "$(workspaces.source.path)/source/scripts/konflux/subscription-manager/subscription-manager-bro.sh" smuggle
 
+    - name: fetch-external-networks
+      runAfter:
+      - clone-repository
+      taskSpec:
+        steps:
+          - name: fetch-external-networks
```

Comment:
My initial idea was to have this download as another step of the existing `fetch-external-content` task.
It is also fine to keep this download separately but in this case I'd ask you to rename `fetch-external-content` to something more specific, like `copy-subscription-manager-activation-key`.

Reply:
Done in https://github.com/stackrox/stackrox/pull/10672/commits/25f21058a2586f79b61718be2ff9b6229e91852d